src/reference/environments/live-basic.yml

```diff
@@ -0,0 +1,9 @@
+name: cylc-8-live-basic
+# Cylc 8 without GUI installed from the default GitHub branch.
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - pip
+  - pip:
+    - git+https://github.com/cylc/cylc-flow.git
```

Comment:
I'm not sure we would want to publicise this approach, except for installing from local working copies.

Reply:
shall I remove these two then @oliver-sanders ?

Reply:
Unless there's a use case for them, probably yes.

Reply:
They've gone from the rst but the yaml files are still there